src/ACFFields.php

```diff
@@ -0,0 +1,138 @@
+<?php
+/**
+ * PHP version 5.6
+ *
+ * @package wpml-to-polylang
+ */
+
+namespace WP_Syntex\WPML_To_Polylang;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Handles the ACF fields.
+ *
+ * @since 0.6
+ */
+class ACFFields extends AbstractSteppable {
+	const ACFML_KEY = 'wpml_cf_preferences';
+	const TYPES     = [
+		0 => 'none', // WPML_IGNORE_CUSTOM_FIELD.
+		1 => 'copy', // WPML_COPY_CUSTOM_FIELD.
+		2 => 'translate', // WPML_TRANSLATE_CUSTOM_FIELD.
+		3 => 'copy_once', // WPML_COPY_ONCE_CUSTOM_FIELD.
+	];
+
+	/**
+	 * Returns the action name.
+	 *
+	 * @since 0.6
+	 *
+	 * @return string
+	 */
+	public function getName() {
+		return 'process_acf_fields';
+	}
+
+	/**
+	 * Returns the processing message.
+	 *
+	 * @since 0.6
+	 *
+	 * @return string
+	 */
+	protected function getMessage() {
+		return esc_html__( 'Processing ACF fields', 'wpml-to-polylang' );
+	}
+
+	/**
+	 * Processes the ACF fields.
+	 *
+	 * @since 0.6
+	 *
+	 * @return void
+	 */
+	protected function handle() {
+		global $wpdb;
+
+		$batch_size = absint( WPML_TO_POLYLANG_QUERY_BATCH_SIZE / 100 ); // 50 by default, to limit the size of the UPDATE query.
+		$offset     = absint( ( $this->step * $batch_size ) - $batch_size );
+		$results    = $wpdb->get_results(
+			$wpdb->prepare(
+				"SELECT ID, post_content
+				FROM {$wpdb->posts}
+				WHERE post_type = 'acf-field' AND post_content LIKE %s
+				ORDER BY ID ASC
+				LIMIT %d, %d",
```

Comment:
```suggestion
				LIMIT %d OFFSET %d",
```
If some day WordPress is kind enough to support PostgreSQL...

Reply:
I didn't know this notation and I find it easier to understand. Done in [4f701b6](https://github.com/polylang/wpml-to-polylang/pull/25/commits/4f701b635f2c14fac1ec15efad50e69d3ddce0a1).